packages/create-platformatic-auto/test/cli/helper.mjs

```diff
@@ -49,11 +49,21 @@ export const getServices = async (dir) => {
 //    match: 'Server listening at',
 //    do: [keys.DOWN, keys.ENTER]
 // }
-export async function executeCreatePlatformatic (dir, actions = [], done = 'All done!', pkgMgrInstall = false) {
+export async function executeCreatePlatformatic (dir, actions = [], done = 'All done!', pkgMgrInstall = false, pkgManager = 'npm') {
   const runCreatePlatformatic = async () => {
     const questions = [...actions]
     try {
-      const child = execa('node', [createPath, `--install=${pkgMgrInstall.toString()}`], { cwd: dir })
+      const execaOptions = {
+        cwd: dir
+      }
+
+      if (pkgManager === 'pnpm') {
+        execaOptions.env = {
+          npm_config_user_agent: 'pnpm/6.14.1 npm/? node/v16.4.2 darwin x64'
```

Comment:
because [this](https://github.com/platformatic/platformatic/blob/main/packages/utils/lib/get-pkg-manager.js) is how we detect if the system runs `pnpm` or other package managers